mldsa/ntt.c

```diff
@@ -17,6 +17,17 @@ __contract__(
   /* TODO: reason about bounds */
 }
 
+static int32_t fqscale(int32_t a)
```

Comment:
Please add a docstring comment here explaining the function and input + output bounds.

Reply:
rename to mld_fqscale